elastalert/elastalert.py

```diff
@@ -197,21 +205,24 @@ def get_index_start(self, index, timestamp_field='@timestamp'):
         """
         query = {'sort': {timestamp_field: {'order': 'asc'}}}
         try:
-            res = self.current_es.search(index=index, size=1, body=query, _source_include=[timestamp_field], ignore_unavailable=True)
+            res = self.current_es.search(index=index, size=1, body=query, _source_include=[timestamp_field] ,ignore_unavailable=True)
```

Comment:
Why was this changed? Please use func(arg, arg2) format instead of func(arg , arg2).